biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -1109,16 +1131,26 @@ subroutine LeafLayerPhotosynthesis(f_sun_lsl,         &  ! in
                  call quadratic_f (aquad, bquad, cquad, r1, r2)
                  agross = min(r1,r2)
 
-                 ! Net carbon assimilation. Exit iteration if an < 0
-                 anet = agross  - lmr
-                 if (anet < 0._r8) then
-                    loop_continue = .false.
-                 end if
+
+                !!! Calculate anet, only exit iteration with negative anet when using anet in calculating gs
+                ! this is version B  
+                 anet = agross - lmr
```

Comment:
I guess I am a bit confused about why this section on agros is in this PR? Is it allied to the modification of gs by LWP, or orthogonal to it? If it's a seperate issue, should it be in a separate PR?

Reply:
Sort of. If using lwp to constrain the transpiration, it is more appropriate to use agross instead of anet